lua/entities/gmod_wire_expression2/core/strfunc.lua

```diff
@@ -84,14 +83,12 @@ registerOperator( "stringcall", "", "", function(self, args)
 
 	local func, func_return_type = findFunc( self, funcname, typeids, typeids_str )
 
-	if not func then E2Lib.raiseException( "No such function: " .. funcname .. "(" .. tps_pretty( typeids_str ) .. ")", 0 ) end
+	if not func then E2Lib.raiseException( "No  such function: " .. funcname .. "(" .. tps_pretty( typeids_str ) .. ")", 0 ) end
```

Comment:
Added a space here on accident? (This would actually break the ``@strict`` unit test afaik)